src/components/CodeHighlight/LogColorizer.css

```diff
@@ -0,0 +1,69 @@
+.toolbar {
+  position:sticky;
```

Comment:
```suggestion
  position: sticky;
```
Please handle other places for consistency.

Reply:
Done